src/components/Explore/TokenDetails/ShareButton.tsx

```diff
@@ -1,83 +1,59 @@
 import { useOnClickOutside } from 'hooks/useOnClickOutside'
-import { darken } from 'polished'
-import { useRef, useState } from 'react'
-import { Check, Link, Share, Twitter } from 'react-feather'
+import { useRef } from 'react'
+import { Twitter } from 'react-feather'
 import { useModalIsOpen, useToggleModal } from 'state/application/hooks'
 import { ApplicationModal } from 'state/application/reducer'
 import styled, { useTheme } from 'styled-components/macro'
-import { Z_INDEX } from 'theme'
+import { ClickableStyle, OPACITY_CLICK, Z_INDEX } from 'theme'
+
+import { ReactComponent as ShareIcon } from '../../../assets/svg/share.svg'
+import { CopyHelper } from '../../../theme'
 
 const TWITTER_WIDTH = 560
 const TWITTER_HEIGHT = 480
 
 const ShareButtonDisplay = styled.div`
   display: flex;
-  cursor: pointer;
   position: relative;
-  z-index: ${Z_INDEX.dropdown};
+`
 
-  &:hover {
-    color: ${({ theme }) => darken(0.1, theme.textSecondary)};
-  }
+const Share = styled(ShareIcon)<{ open: boolean }>`
+  fill: ${({ theme }) => theme.textSecondary} !important;
+  ${ClickableStyle}
+  ${({ open }) => open && `opacity: ${OPACITY_CLICK} !important`};
 `
+
 const ShareActions = styled.div`
   position: absolute;
+  z-index: ${Z_INDEX.dropdown};
+  width: 240px;
   top: 28px;
   right: 0px;
-  padding: 8px 0px;
+  justify-content: center;
   display: flex;
   flex-direction: column;
-  width: fit-content;
   overflow: auto;
+  padding: 8px;
   background-color: ${({ theme }) => theme.backgroundSurface};
-  border: 1px solid ${({ theme }) => theme.backgroundOutline};
+  border: 0.5px solid ${({ theme }) => theme.backgroundOutline};
   box-shadow: ${({ theme }) => theme.flyoutDropShadow};
   border-radius: 12px;
 `
-const ShareAction = styled.div`
+const ShareAction = styled.div<{ highlighted?: boolean }>`
+  ${({ highlighted }) => highlighted && ClickableStyle};
   display: flex;
   align-items: center;
-  padding: 12px 16px;
+  padding: 8px;
+  border-radius: 8px;
   font-size: 16px;
-  gap: 8px;
-  width: 200px;
-  height: 48px;
+  font-weight: 400;
+  gap: 12px;
+  height: 40px;
   color: ${({ theme }) => theme.textPrimary};
-  cursor: pointer;
-
-  &:hover {
-    background-color: ${({ theme }) => theme.backgroundModule};
-  }
+  background-color: ${({ theme, highlighted }) => (highlighted ? theme.backgroundInteractive : 'transparent')};
+  ${({ highlighted }) => highlighted && `cursor: pointer`};
```

Comment:
can't we always just have this as cursor pointer since anytime we hover over an option it should be a pointer? regardless of whether it's highlighted or not?

Reply:
I made it so that it wouldn't show the pointer if you weren't actually hovering over the copy helper, but since the whole button is supposed to change bgcolor I'm gonna revert